packages/core/src/components/SortableTable/SortableTable.tsx

```diff
@@ -68,25 +100,15 @@ const EnhancedTableHead = ({ columns, onRequestSort, order, orderBy }) => {
   );
 };
 
-EnhancedTableHead.propTypes = {
-  columns: PropTypes.arrayOf(
-    PropTypes.shape({
-      id: PropTypes.string.isRequired,
-      label: PropTypes.string.isRequired,
-      numeric: PropTypes.bool,
-      disablePadding: PropTypes.bool,
-      style: PropTypes.object,
-    }),
-  ).isRequired,
-  onRequestSort: PropTypes.func.isRequired,
-  order: PropTypes.string.isRequired,
-  orderBy: PropTypes.string.isRequired,
+type CellContentProps = {
+  // XXX (@Koroeskohr): what am I supposed to use here
+  data: any | any[];
```

Comment:
https://github.com/typescript-cheatsheets/react-typescript-cheatsheet/blob/cc2ec67e0b36b7e838df8f2a54b4cd84df6c78cd/README.md#useful-react-prop-type-examples
`React.ReactNode` would probably be the best, seeing from the context